loguru.hpp

```diff
@@ -1759,11 +1842,21 @@ namespace loguru
 
 		const char* mode_str = (mode == FileMode::Truncate ? "w" : "a");
 		auto file = fopen(path, mode_str);
+#ifdef LOGURU_WITH_FILEABS
+		FileAbs* file_abs = new FileAbs(); // this is deleted file_close;
```

Comment:
Uh, ``file_log`` etc uses ``void* user_data`` as a parameter. I don't know how to do better.